platypus/parse.py

```diff
@@ -141,12 +136,22 @@ def process_results(percentage_ids, alignment_lengths, percentage_ids_other,
     len_alignment_lengths = len(alignment_lengths)
     results = []
 
-    for i, j in [(i,j) for i in range(len_percentage_ids) for j in range(len_alignment_lengths)]:
-        filename = "p1_%d-a1_%d_p2_%d-a2_%d" % (percentage_ids[i],
-            alignment_lengths[j], percentage_ids_other[i], alignment_lengths_other[j])
-        results.append({ 'filename': filename, 'db_interest': 0, 'db_other': 0,
-            'perfect_interest': 0, 'equal': 0, 'summary': ['#SeqId\tFirst\t'
-            'Second'], 'db_seqs_counts': {'a': {}, 'b': {} } })
+    values = [(i, j) for i in range(len_percentage_ids) for j in
```

Comment:
...but I think this can be done over `percentage_ids` with something like:

```python
iter_a = product(percentage_ids, alignment_lengths)
iter_b = product(percentage_ids_other, alignment_lengths_other)
for (perc_id_a, aln_len_a), (perc_id_b, aln_len_b) in izip(iter_a, iter_b):
    # ...
```

This assumes lengths of the lists going in are the same etc. The order should be deterministic for `product` as well